setup.py

```diff
@@ -25,5 +25,9 @@
     packages=["pyspline"],
     package_data={"pyspline": ["*.so"]},
     install_requires=["numpy>=1.16", "scipy>=1.2"],
+    extra_requires={
+        "docs": ["sphinx-mdolab-theme"],
```

Comment:
I managed to unify the two documentation requirements by parsing `doc/requirements.txt` [here](https://github.com/mdolab/prefoil/pull/51/commits/463ce713b3f4e7be45c9cd89ab669fd818c2cc63). I would prefer this approach to what's done here